src/utils/helpers.ts

```diff
@@ -290,6 +290,11 @@ export const calculateAssetValue = (
   return numberFromWei(bignumber(amount).mul(rate));
 };
 
+const areAssetsEqual = (asset1: Asset, asset2: Asset) =>
```

Comment:
```suggestion
const areBtcAssets = (asset1: Asset, asset2: Asset) =>
```

Reply:
Why `areBtcAssets`? Function verifies if asset1==asset2 and additionally checks if it's WRBTC & RBTC (because it's same thing on different contracts)

Reply:
Sorry, I overlooked it and thought it was just checking if are both assets `WRBTC` or `RBTC` :D.